astropy/cosmology/core.py

```diff
@@ -777,6 +791,46 @@ def nu_relative_density(self, z):
 
         return prefac * self._neff_per_nu * rel_mass
 
+    def _get_nufunc_scalar(self):
+        """ Return the version of the nu relative density function
+        to use in integrals."""
+
+        if self._massivenu:
+            return self._massivenu_relative_density_scalar
+        else:
+            return lambda x : 0.22710731766 * self._Neff
```

Comment:
If you are doing serious cosmography these days, you almost have to use some version of CosmoMC.  Which is in Fortran.  I did have to write a module for it a few years ago, which required me to have a Fortran book open in front of me the entire time.